exosphere-shared/example-apps/tests-failing/tweets-service/service.yml

```diff
@@ -1,5 +1,6 @@
-title: another service with tests
+title: another service with tests failing
```

Comment:
Let's say "with failing tests" instead of "with tests failing". Please change the other files as well.